srcpkgs/python3-dotty-dict/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'python3-dotty-dict'
+pkgname=python3-dotty-dict
+version=1.3.0
+revision=1
+wrksrc="dotty_dict-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+depends="python3"
+short_desc="Dictionary wrapper for quick access to deeply nested keys"
+maintainer="Orphaned <orphan@voidlinux.org>"
```

Comment:
We usually don't add orphaned packages, would you like to assign yourself as their maintainer?

Reply:
I really have no desire to maintain python micro dependencies. I'm only doing this because I wanted to use `qmk` and it was out of date. I have no intention of maintaining these python packages so I thought marking them as orphans would be best.

If it's a showstopper I can put name on them, but I don't intend to ever touch them again if I can help it.